src/main.js

```diff
@@ -21,7 +21,7 @@ pkg.initFormat();
 pkg.require({
     Gio: '2.0',
     Gtk: '3.0',
-    Handy: '0.0',
+    Handy: '1',
```

Comment:
_1_, not _1.0_?

Reply:
No because that's not the project version but the API version, 0.0 should have been 0 in the first place. :)

Reply:
I mean, it should have been 0 on libhandy's side when we started the project. regarding Flatseal, 0.0 was right and 1 is right. :)